packages/docs/src/components/DocsNavigation.vue

```diff
@@ -51,66 +65,69 @@ const query = '?query=file=/src/App.vue'
 
 type ButtonStates = 'active' | 'error' | 'default'
 
-export default defineComponent({
-  name: 'DocsNavigation',
-  props: {
-    code: { type: String as PropType<string>, default: '' },
-    config: { type: Object as PropType<CodesandboxConfig>, default: () => ({}) },
-    gitUrl: { type: String as PropType<string>, default: '' },
-    gitComponent: { type: String as PropType<string>, default: '' },
-  },
-  setup (props) {
-    const { t } = useI18n()
-
-    const copyButtonState: Ref<ButtonStates> = ref('default')
-    const copy = async () => {
-      try {
-        await getWindow()?.navigator.clipboard.writeText(props.code)
-        copyButtonState.value = 'active'
-      } catch (e: any) {
-        if (e.message === 'NotAllowedError') {
-          copyButtonState.value = 'error'
-        }
-      }
-
-      setTimeout(() => { copyButtonState.value = 'default' }, 1500)
-    }
+const props = defineProps({
+  code: { type: String as PropType<string>, default: '' },
```

Comment:
As I remember we decided to not use the `PropType`, is it necessary in this property and other ones?

Reply:
Yeah. We removed them from UI lib, but not docs.

Reply:
Probably we want to remove from both places at once. (maybe as a separate issue?)